modules/browser/hooked_domain/apache_tomcat_examples_cookie_disclosure/config.yaml

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2006-2021 Wade Alcorn - wade@bindshell.net
+# Browser Exploitation Framework (BeEF) - http://beefproject.com
+# See the file 'doc/COPYING' for copying permission
+#
+beef:
+    module:
+      apache_tomcat_examples_cookie_disclosure:
+            enable: true
+            category: "Exploits"
```

Comment:
Only comment in the steps you've got 'Run the Apache Tomcat RequestHeaderExample Cookie Disclosure module from the Browser -> Hooked Domain category'
in the configuration it's listed a Exploits

Reply:
Up to you. The `modules/exploits/apache_cookie_disclosure/` is similar and exists in the exploit category.

Reply:
Changed to hooked browser category.